src/AM43Client.h

```diff
@@ -53,6 +53,8 @@ class AM43Client : public BLEClientCallbacks {
     boolean m_DoConnect;
     // We're logged in (correct pin)
     boolean m_LoggedIn;
+    // OnDemand command flag
+    bollean m_CommandQueued;
```

Comment:
Please fix the misprint:  boolean instead of bollean

Reply:
Ok, done!